workspaces/marketplace/plugins/marketplace/src/utils.ts

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import jsyaml from 'js-yaml';
+import { Pair, parseDocument, Scalar } from 'yaml';
+import { get } from 'lodash';
+import { JsonObject } from '@backstage/types';
+
+export const getYamlContent = (content: JsonObject) => {
+  const yamlString = jsyaml.dump(content);
+  return `\`\`\`yaml\n${yamlString}\n\`\`\``;
+};
+
+export const applyContent = (
+  editorContent: string,
+  newContent: string,
+  packageName: string,
+) => {
+  const content = parseDocument(editorContent);
+  const plugins = get(content.get('plugins'), 'items') ?? [];
```

Comment:
Do we need the extra lodash dependency for this one usage? 

Since lodash hasn't been updated for a while I am hesitant to add it